torch/csrc/jit/codegen/cuda/kernel_ir.h

```diff
@@ -452,6 +452,10 @@ class TORCH_CUDA_CU_API ForLoop final : public Expr {
 
   Val* step() const;
 
+  Val* simplifiedStart() const;
```

Comment:
Do we need both the original `start` and `simplifiedStart`?

Reply:
Actually, I guess just the original `start` should be fine? I have never seen a nontrivial start.

Reply:
And, you think just the `simplifiedStart` is fine?

Reply:
Yeah, just `simplifiedStart` is fine.

Reply:
I believe it should just be the former.

We may need to support dynamic shift, where a start value can be a symbolic value, but that's not what's supported now, and there's no concrete plan for dynamic shift.

Reply:
Also agree with remove `simplifiedStart` (and also `simplifiedStop`)

Reply:
OK, I will remove `simplifiedStart` and change back to `start`. But I have to keep both `stop` and `simplifiedStop` because there are checks like `if (stop() == id->extent())` which would be broken with `simplifiedStop`...

Reply:
removed `simplifiedStart`

Reply:
Oh, I meant to keep `simpifiedStart` but rename it as just `start`. Sorry being not clear. I suppose it doesn't matter much anyway.